custom/CustomCAN.java

```diff
@@ -62,7 +63,9 @@ protected ByteBuffer readBuffer() {
 		try {
 			response = CANJNI.FRCNetCommCANSessionMuxReceiveMessage(idBuffer, 0x1fffffff, timestamp);
 		}
-		catch (CANMessageNotFoundException e) {}
+		catch (CANMessageNotFoundException e) {
+			LogKitten.w(e.getMessage()); // We are not actually sure when this error is used (no docs), so just warn for now
```

Comment:
Actually, I think the way I am doing this now is entirely wrong. Without the loop, there is no reason not to pass on the error.